themes/custom/droopler_theme/scss/components/_d_p_banner.scss

```diff
@@ -31,41 +31,45 @@ $d-p-banner-subheader-opacity: 0.7 !default;
 
 // Override default values.
 .padding-top-default {
-  .paragraph--type--d-p-banner {
-    padding-top: 0;
-    .content-wrapper {
-      padding-top: $d-p-banner-padding-top;
-      @include media-breakpoint-down(sm) {
-        padding-top: $d-p-banner-mobile-padding-top;
-      }
+  .paragraph.paragraph--type--d-p-banner {
+    padding-top: $d-p-banner-padding-top;
+
+    @include media-breakpoint-down(sm) {
```

Comment:
nie mamy mobile first?

Reply:
Skórka nie była tak projektowana, a na refactor trochę za późno. Także robimy tak, żeby było optymalnie, bo konwencja nie została ustalona.

Reply:
true,
ale akurat tutaj moge to zmienic.